crossing-training.lic

```diff
@@ -192,6 +194,16 @@ class CrossingTraining
       pause 15
     end
   end
+  
+  def refresh_cyclic
+    return unless @settings.cyclic_training_spells
+    if @settings.cyclic_training_spells && (Time.now - @cyclic_cycle_timer > 300)
+      cyclic_skill = @settings.cyclic_cycle_skills
```

Comment:
as a note - this won't stop you from always having a cyclic up - that's actually something crossing-training already does if you train with cyclics. There are limited times it calls to release_cyclic and typically it's done just before casting a new cyclic spell. If you stop working on magic because it is all locked, release cyclics wont be called. I can add a feature to stop all cyclics in cyclic cycle when over an XP threshold though if that's what we are interested in.